eks/sync.go

```diff
@@ -250,37 +255,47 @@ func upgradeCoreDNS(
 ) error {
 	logger := logging.GetProjectLogger()
 
-	targetImage := fmt.Sprintf("602401143452.dkr.ecr.%s.amazonaws.com/eks/coredns:v%s", awsRegion, coreDNSVersion)
-	currentImage, err := getCurrentDeployedCoreDNSImage(clientset)
-	if err != nil {
-		return err
-	}
-	if currentImage == targetImage {
-		logger.Info("Current deployed version matches expected version. Skipping coredns update.")
-		return nil
-	}
-
 	logger.Info("Confirming compatibility of coredns configuration with latest version.")
-	corednsConfigMap, err := getCorednsConfigMap(clientset)
-	if err != nil {
-		return err
-	}
 	// Need to check config for backwards incompatibility if updating to version >= 1.7.0. The keyword upstream was
```

Comment:
NIT: I would probably word this comment as 
> The keyword "upstream" or \`upstream\`

to be clear that `upstream` is the keyword.

Reply:
Done! 7cb60c0